code/modules/mob/living/living_defense.dm

```diff
@@ -3,7 +3,7 @@
 	var/armor = getarmor(def_zone, attack_flag)
 
 	//the if "armor" check is because this is used for everything on /living, including humans
-	if(armor && armour_penetration)
+	if(armor > 0 && armour_penetration)
 		armor = max(0, armor - armour_penetration)

```

Comment:
regardless of the amount of armor, there will always be more or equal to 0

Reply:
or you want negative armor to increase damage?

Reply:
if you want to make armor that increases damage, better remove this string and add message like ,"your armor makes it worse"

Reply:
Negative armor is supposed to increase damage. It already does but if you attack with armor penetration the negative armor gets increased to 0 meaning less damage gets dealt.